docs/changelog.md

```diff
@@ -1,5 +1,52 @@
 ## Changelog
 
+
+### v0.17.0 (1/9/2021)
+
+#### Compatibility
+
+- Unify the camera keys for consistent transformation between coodinate systems on different datasets. The modification change the key names to `lidar2img`, `depth2img`, `cam2img`, etc. for easier understanding. Customized codes using legacy keys may be influenced.
+- The next release will begin to move cuda op files to [mmcv](https://github.com/open-mmlab/mmcv). It will influence the way to import related functions. Please stay tuned.
+
+#### Highlights
+
+- Support a new task, 3D object detection, on the S3DIS dataset
```

Comment:
Support a new task, 3D object detection, on the S3DIS dataset -> Support 3D object detection on the S3DIS dataset